closure/BUILD

```diff
@@ -38,13 +38,6 @@ lines_sorted_test(
     file = "repositories.bzl",
 )
 
-lines_sorted_test(
```

Comment:
Why is this removed ?

Reply:
The sorting it was enforcing wasn't conforming to sorting done by buildifier.